ITISFoundation/osparc-simcore
# 🐛 Fix opentelemetry errors in autoscaling microservice

## What do these changes do?
Connect autoscaling microservice to `default` network in local dev-deploy. Enables autoscaling to communicate with the opentelemetry collector.


## Related issue/s




## How to test


1. `make build up-prod`
2. Run `docker service logs master-simcore_autoscaling`, see no more opentelemetry errors
## Dev-ops checklist
